### PR TITLE
Video: Send VIDEO_STREAM_INFORMATION to GCS

### PR DESCRIPTION
**Pull request by ddd999**
Addresses feature request issue #169

Enables Rpanion to respond to send COMMAND_ACK and VIDEO_STREAM_INFORMATION messages via MavLink in response to a MAV_CMD_REQUEST_MESSAGE.

### CODE REVIEW

**Comment by ddd999** [comment]
Converting to draft because I realized it's going to be mandatory to have a way to select the network interface (and therefore, which URI gets send in the VIDEO_STREAM_INFORMATION) message. This would work fine hardcoded, but there are plenty of use cases where a user would want to choose.

**Comment by stephendade** [comment]
Nice! When you're ready, happy to give it test and review.

**Comment by ddd999** [comment]
@stephendade, please review when you have time. Thank you!

Thanks to @TyIsI for helping with tips to get this working as well.

**Comment by stephendade** [comment]
Great! Looks like you'll need a rebase first, as some of my recent tlogging changes aren't in here.

**Comment by ddd999** [comment]
@stephendade I had a bit of fight with git but I believe it is all up to date now.

**Comment by stephendade** [comment]
Yes, that look up to date now. I should have some time in the next few days to review.

**Review by stephendade — CHANGES_REQUESTED** [review]
Looks pretty good. Some minor formatting issues and the user needs a bit more information about the service.

File: src/video.js [hunk 1]
```diff
@@ -186,6 +194,12 @@ class VideoPage extends basePage {
             <input disabled={this.state.streamingStatus} type="number" name="fps" min="1" max={this.state.FPSMax} step="1" onChange={this.handleFPSChange} value={this.state.fpsSelected} />fps (max: {this.state.FPSMax})
           </div>
         </div>
+        <div className="form-group row" style={{ marginBottom: '5px' }}>
```
Comment (stephendade) [inline src/video.js:197]: <br />
        <h3>MAVLink Video Streaming Service</h3>
        <p><i>Configuration for advertising the video stream via MAVLink. See <a href='https://mavlink.io/en/services/camera.html#video_streaming'>here</a> for details.</i></p>
        <div className="form-group row" style={{ marginBottom: '5px' }}>
          <label className="col-sm-4 col-form-label">Video source IP Address</label>
          <div className="col-sm-8">
            <Select isDisabled={this.state.streamingStatus} onChange={this.handleMavStreamChange} options={this.state.ifaces.map((item, index) => ({ value: index, label: item}))} value={this.state.mavStreamSelected} />
          </div>
        </div>

Comment (stephendade) [reply src/video.js:197]: Also needs the "Stream Configuration" header changed to h3 to match here

File: mavlink/mavManager.js [hunk 2]
```diff
@@ -100,6 +100,14 @@ class mavManager {
 
         // send off initial messages
         this.sendVersionRequest()
+
+        // Respond to MavLink commands that are targeted to the companion computer
+      } else if ( data.targetSystem === this.targetSystem && 
+        data.targetComponent == minimal.MavComponent.ONBOARD_COMPUTER &&
+        packet.header.msgid === common.CommandLong.MSG_ID
+      ){
```
Comment (stephendade) [inline mavlink/mavManager.js:108]: Space between brackets

File: mavlink/mavManager.js [hunk 3]
```diff
@@ -100,6 +100,14 @@ class mavManager {
 
         // send off initial messages
         this.sendVersionRequest()
+
+        // Respond to MavLink commands that are targeted to the companion computer
```
Comment (stephendade) [inline mavlink/mavManager.js:104]: Remove any extraneous trailing whitespaces from this section

**Review by stephendade — CHANGES_REQUESTED** [review]

**Comment by stephendade** [comment]
Looks good! Merging...

**Pull request merged**